### PR TITLE
Add all speech notifications to notifications.yaml

### PR DESCRIPTION
**Pull request by abcdefg30**
And also make use of them. Followups are on their way.

Depends on https://github.com/OpenRA/OpenRA/pull/10564 at the moment.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Rebased as OpenRA/OpenRA#10564 was merged.

File: audio/notifications.yaml [hunk 1]
```diff
@@ -11,59 +11,122 @@ Speech:
...
 		Building: 052
 		BuildingCannotPlaceAudio: 063
 		BuildingCaptured: 072
 		BuildingInfiltrated: 073
+		BuildingInfiltratedCashStolen: 090
+		BuildingInfiltratedPowerSabotaged: 091
+		BuildingInfiltratedRadarSabotaged: 089
+		BuildingInfiltratedTechnologyStolen: 088
+		BuildingOffline: 060
+		BuildingOnline: 061
```
Comment (Mailaender) [inline audio/notifications.yaml]: These seem to be flipped.

Comment (abcdefg30) [inline audio/notifications.yaml]: Derp, fixed.

**Comment by abcdefg30** [comment]
Updated.

**Comment by Mailaender** [comment]
Thanks!

**Pull request merged**